plugins/modules/ipaautomountmap.py

```diff
@@ -104,7 +104,10 @@ def get_automountmap(self, location, name):
         except Exception:  # pylint: disable=broad-except
             return None
         else:
-            return response["result"]
+            _result = response["result"]
+            if 'description' not in _result:
+                _result['description'] = ''
+            return _result
```

Comment:
Do we have the description issue in more modules? It might more sense to handle a comparison of the empty string in comapre_args_ipa. What do you think?

Reply:
I haven't seen this happen in different modules, but I agree that this should be fixed in `compare_args_ipa`.

Reply:
Comparison has been changed to `module_utils.ansible_freeipa_module.compare_args_ipa`.